style.css

```diff
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+  padding: 0;
+  text-align: center;
```

Comment:
As you can see in the form, you should give a text-align of center to the stuff that should be centered

Reply:
- [ ] Most time for clarity sake, we always want to give reason(s) for required change(s). Can you please state the reason for this change? Some things maybe obvious to you but others may not notice is.👍